mite/cli/collector.py

```diff
@@ -0,0 +1,25 @@
+import asyncio
+
+from ..collector import Collector
+from ..utils import _msg_backend_module, spec_import
+
+
+def _collector_receiver(opts):
+    socket = opts["--collector-socket"]
+    receiver = _msg_backend_module(opts).Receiver()
+    receiver.connect(socket)
+    return receiver
+
+
+def collector(opts):
+    receiver = _collector_receiver(opts)
+    filter_fn = None
+    if filter_fn_spec := opts["--collector-filter"]:
```

Comment:
🙊

Reply:
Ah rats...I guess we're keeping python 3.7 compatibility in mite for now...